iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -1013,7 +1015,11 @@ private void mergeProperties(DanglingLine dl1, DanglingLine dl2, Properties prop
                 LOGGER.warn("Inconsistencies of property '{}' between both sides of merged line. Side 2 is empty, keeping side 1 value '{}'", prop, dl1.getProperty(prop));
                 properties.setProperty(prop, dl1.getProperty(prop));
             } else {
-                LOGGER.error("Inconsistencies of property '{}' between both sides of merged line. '{}' on side 1 and '{}' on side 2. Removing the property of merged line", prop, dl1.getProperty(prop), dl2.getProperty(prop));
+                if (!prop.equals(ELEMENT_NAME_PROPERTY)) {
+                    LOGGER.error("Inconsistencies of property '{}' between both sides of merged line. '{}' on side 1 and '{}' on side 2. Removing the property of merged line", prop, dl1.getProperty(prop), dl2.getProperty(prop));
+                } else {
+                    properties.setProperty(prop, String.format("%s,%s", dl1.getProperty(prop), dl2.getProperty(prop)));
+                }
```

Comment:
I'm not sure we should make a specific case for element names. Maybe we should decide the strategy, and apply it to all properties.
@murgeyseb What do you think?

Reply:
Yes, I think we could keep the same behaviour for Element name, and change the parsing of the property in the exporter to create a unique element name if needed, using the same strategy as before (if equals use it, if one empty use the other, otherwise no element name)

Reply:
But if we don't make a specific case for element names this strategy of concatenating the two  side's property will impcact all types of property. for example the geographical name it will create other problems. Should i delete this condition here ? but make another one in exporter saying if geographival name contains the separator "," --> don't export it